### ISSUE TITLE
Estimated approach time / landing rate indicator

### ISSUE THREAD

**Issue by AndyTWF** [VATSIM-UK/uk-controller-plugin#453]

**Comment by hsugden** [comment]
Can we discuss this please before any work is done on this? Think it would be worth discussing what's actually useful here, as an EAT generator is not very imo :)

**Comment by AndyTWF** [comment]
Yep sure - just an idea that was passed on but more than happy to talk scope :)

**Comment by kristiankunc** [comment]
@hazzas-99 Is this still relevant given TopSky has some prediction functionality (or at least I think so)
